pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
```

Comment:
According to [this](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html) we may need to convert `[tool.poetry]` to `[project]` which is a standard field since [PEP 621](https://peps.python.org/pep-0621/)

Reply:
Oh yes sorry. I completely missed that

Reply:
I really wonder why poetry's pyproject.toml is so far away from the pep standard. But now this should be compliant

Reply:
From what I gathered, poetry started using the pyproject.toml before the project tag was standardized.

Reply:
Ah that would make sense